src/server/master_server/master_chunk_server_manager_service_impl.cc

```diff
@@ -45,6 +45,12 @@ grpc::Status MasterChunkServerManagerServiceImpl::ReportChunkServer(
     std::shared_ptr<protos::ChunkServer> new_chunk_server(
         new protos::ChunkServer(new_server_info));
 
+    // Resolve name if this is necessary
+    if (resolve_hostname_) {
+      new_chunk_server->mutable_location()->set_server_hostname(
```

Comment:
why? name is "hostname" so I will assume it's meant to store host name and not ip address. Services that use this will do the translation to ipaddress

Reply:
Ok, I could leave the resolve hostname on the master side for this. I will revert this part back then.